Attendance/src/main/java/com/system/Attendance/domain/Event.java

```diff
@@ -41,6 +41,8 @@ public class Event implements Serializable {
             joinColumns = {@JoinColumn(name = "event_id")},
             inverseJoinColumns = {@JoinColumn(name = "member_id")})
     private List<Member> memberList = new ArrayList<Member>();
+    @OneToOne
+    private Scanner scanner;
```

Comment:
why you add new relation, scanner has relation with session to get records of sessions

Reply:
I need to be able to fetch scanner details from session and not the other way round